CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added option to flip vertical orientation of checkpoint files
```

Comment:
You might want a bit more documentation here so that users that want to use this feature know how to use it.